src/Microsoft.ML.Core/Prediction/ScoringUtils.cs

```diff
@@ -0,0 +1,71 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using Microsoft.ML.Core.Data;
+using Microsoft.ML.Runtime.Data;
+
+namespace Microsoft.ML.Runtime.Internal.Utilities
+{
+    public static class ScoringUtils
+    {
+        /// <summary>
+        /// Utility function that returns the SchemaShape of the generated outputcolumns when scoring a model.
+        /// </summary>
+        /// <param name="predictionKind">The learning task that is being performed.</param>
+        /// <param name="isNormalized">Whether the probability column (if generated) will be normalized or not.</param>
+        /// <param name="labelColumn"></param>
+        /// <returns></returns>
+        public static SchemaShape GetPrdictorOutputColumns(PredictionKind predictionKind, bool isNormalized = true, SchemaShape.Column? labelColumn = null)
```

Comment:
>Prd [](start = 37, length = 3)

Grt nm.
I mean:
Great name.

Reply:
Thank you ;)

---
In reply to: [240798257](https://github.com/dotnet/machinelearning/pull/1847#discussion_r240798257) [](ancestors = 240798257)